### ISSUE TITLE
Understanding 1.4.10: "In brief" knows only text and horizontal scrolling (i18n)

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#3385]
[In brief](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html#brief): 

> Objective: Content can be enlarged without requiring horizontal scrolling

... or without vertical scrolling

> Author task: Make text reflow within the width of the viewport 

* ... or the heigth of the viewport
* 1.4.8 restricts the requirement to text, but not 1.4.10, so my question would be whether it is presented here in too limited a way or SC 1.4.10 is worded too broadly (https://github.com/w3c/wcag/issues/3378)

**Comment by alastc** [comment]
I think this is simply due to trying to create a very short and simple sentence. 

There is a question around the internationalisation though, looping in @mbgower for his thoughts...

**Comment by bruce-usab** [comment]
Definitely some tension comes with keeping In Brief short and simple.  Maybe:

> Objective: Content can be enlarged without requiring horizontal scrolling (for languages written left-to-right).

The "without horizonal scrolling" is the simplified core concept that folks commonly miss from their first reading of the SC text.

**Comment by mbgower** [comment]
@JAWS-test it's not as simple as saying 'without horizontal or vertical scrolling' because of course that means that you can't scroll at all!

I think it's important to point out that this did just pass a full review by the working group a couple of months ago, where this very topic was debated for some time...

At one point I proposed "sideways" which I _think_ should work in both cases, based on the context of the reader, but few seemed to like that as much as I did. 

Given the SC talks about both horizontal AND vertical scrolling, I agree it's a _challenge_. My brief investigation suggests there are _very_ few languages in the world that are exclusively written vertically, and if we ever had translations of WCAG for those languages, we could simply 'flip' the language.

 If I was going back to this SC now, I would advocate for using "horizontally" in the normative language and address vertical languages (or any alphabet laid out on the vertical plain) in a note. But unfortunately I have not yet mastered overcoming the space-time continuum.

BTW, since I'm looking at the 2.1 In Brief wording in a [currrent PR](https://github.com/w3c/wcag/pull/3312) where I've added in the 'why'. I'll point out that there's a less glaring but parallel problem with "width" in the What line, if we ARE going to try to address this.

Bruce has already made the point, but remember this is a brief summary. We spend a LOT of time on this in the Understanding document, and an outcome of brevity tends to be imprecision.

That said, let's revisit "sideways" for use in "in brief".  I see it **is** used once in the Understanding doc, so we have our butts covered.
Anyone think this is better in a brief description?

Current
<dt>Goal</dt><dd>Content can be enlarged without requiring horizontal scrolling.</dd>
				<dt>What to do</dt><dd>Make text reflow within the width of the viewport.</dd>
	      <dt>Why it's important</dt><dd>People who need larger text find it difficult to read long lines of text.</dd>

Suggested
<dt>Goal</dt><dd>Content can be enlarged without requiring sideways scrolling.</dd>
				<dt>What to do</dt><dd>Make text reflow within the viewport.</dd>
	      <dt>Why it's important</dt><dd>People who need larger text find it difficult to read long lines of text.</dd>

I think that solves both problems. Potentially someone could argue "within the viewport" is TOO constraining, but I think we _may_ be over critiquing.

**Comment by GreggVan** [comment]
Since this says   Horiz OR vertical  and not   Horiz  AND vertical  —  I think the issued does not arise (though that subtlety should be explained in the Understanding doc)

Also    "without horizontal was meant for  left-right(or right-left) text and vertical was meant for  Top/down text.  

Clearly we need to do a better job of this in WCAG 3.0.    I think for  2.2   (since it is in the can)  we should fix it in Understanding. 

ALSO  the reason for the no-horizontal scrolling is not "people have trouble with long lines" exactly 
I was told it was actually   "People who have to horizontally scroll have difficulty tracking on the line when scrolling and also finding the ’next line’ when they scroll back."   


gregg

———————————
Professor, University of Maryland, College Park
Founder and Director Emeritus , Trace R&D Center, UMD
Co-Founder Raising the Floor. http://raisingthefloor.org
The Global Public Inclusive Infrastructure (GPII) http://GPII.net
The Morphic project  https://morphic.org

**Comment by mbgower** [comment]
> People who have to horizontally scroll have difficulty tracking on the line when scrolling and also finding the ’next line’ when they scroll back."

That notion seems to me to be fairly captured in the phrase "people have trouble with long lines"

The Understanding document currently says:
> Reflow enables tracking. Tracking is following along lines of text, including getting from the end of one line to the beginning of the next line.

We can always build on this existing text. I have now referenced this issue in the Issue created to do updates to Understanding documents based on the "in brief" material.

**Comment by mbgower** [comment]
Actually, I took another stab at this today, and I think I may have arrived at something that works better. How about:
><dt>Goal</dt><dd>Content can be enlarged without affecting line length.</dd>
><dt>What to do</dt><dd>Make lines of text reflow within the viewport.</dd>
><dt>Why it's important</dt><dd>People who need larger text find it difficult to read long lines of text.</dd>

**Comment by detlevhfischer** [comment]
@mbgower That's confusing IMO - reflow does exactly that, affect line length? Not semantically perhaps, but visually.

**Comment by mbgower** [comment]
I've actually accepted those changes. This is how it nows reads:
<dt>Goal</dt><dd>Content can be enlarged without increasing line length.</dd>
				<dt>What to do</dt><dd>Make lines of text reflow within the viewport.</dd>
	      <dt>Why it's important</dt><dd>People who need bigger text find it difficult if they must scroll to read long lines.</dd>

**Comment by mbgower** [comment]
@JAWS-test The In Brief text was updated so that it no longer refers to direction.
Please see https://www.w3.org/WAI/WCAG22/Understanding/reflow.html
If you feel your concern was not addressed by the update, please reopen and add comments.

**Comment by A11yTea** [comment]
Hey I think this issue needs to be reopened. The In Brief leads me to a different interpretation of 1.4.10 Reflow. Here are my comments on each part:
**Goal**
Content can be enlarged without increasing line length.
**Comment on Goal:** I have no idea what this means, and how do I look for line length? What does line length mean?
**What to do**
Make lines of text reflow within the viewport.
**Comment on What to do:** So non-text content are exempt? Like inputs dont need to reflow? Why does the criteria use the term "content" and the in brief is not?
**Why it's important**
People who need bigger text find it difficult if they must scroll to read long lines.
**Comment on Why it's important:** I think that is okay

**Comment by scottaohara** [comment]
@A11yTea i'd suggest you create a new issue. while being about the in brief content, your comments go beyond what this issue was originally about.

**Comment by A11yTea** [comment]
Will do!
